docs/source/tutorials/stable_diffusion.mdx

```diff
@@ -56,7 +56,7 @@ outputs = pipeline(
 )
 ```
 
-Outputs can be PIL images or Numpy arrays. See [here](https://huggingface.co/docs/optimum/habana/package_reference/stable_diffusion_pipeline#optimum.habana.diffusers.StableDiffusionPipeline.__call__) all the parameters you can set to tailor generations to your taste.
+Outputs can be PIL images or Numpy arrays. See `StableDiffusionPipeline` all the parameters you can set to tailor generations to your taste.
```

Comment:
In the second sentence, either we keep the link either we rephrase it but the current version doesn't mean anything IMO.

Reply:
Done